app/views/home/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="row">
+<% @photos.each do |photo| %>
+      <% counter = 0 %>
+      <% if photo.parent_uuid != nil %>
+        <% counter += 1 %>
+        <% if counter % 4 == 0 %>
+          <div = class="row">
+        <% end %>
+        <div class="col-xs-6 col-md-3">
+            <img src='<%= CloudStorage::Object.new(photo.object_key).url %>' class="img-thumbnail"/>
```

Comment:
Be consistent with HTML attributes single/double quotes. In general, double quotes is more popular and widely used.